Cargo.toml

```diff
@@ -86,5 +86,5 @@ codegen-units = 1
 # TODO: Remove once chacha20poly1305 0.10 appears in libp2p's dependencies
 chacha20poly1305 = { git = "https://github.com/RustCrypto/AEADs", rev = "06dbfb5571687fd1bbe9d3c9b2193a1ba17f8e99" }
 libp2p = { git = "https://github.com/subspace/rust-libp2p", branch = "subspace-v3" }
-# TODO: Remove once new version of parity-db will be released
-parity-db = { git = "https://github.com/paritytech/parity-db/", rev = "5c6abf17e0027935de9c78959c73b8c35835cc10" }
+# TODO: Remove once parity-db 0.3.17 will be released
+parity-db = { git = "https://github.com/i1i1/parity-db/", rev = "b2f2a057fbf6f0c7e3b24e3fda35879dc76c7d84" }
```

Comment:
Please push code changes to subspace org if you need to reference it here

Reply:
Do you mean that we could for `parity-db` inder subspace domain?